src/app.service.ts

```diff
@@ -3,6 +3,6 @@ import { Injectable } from '@nestjs/common';
 @Injectable()
 export class AppService {
   getHello(): string {
-    return 'Hello World!';
+    return 'Hello World stuped!';
```

Comment:
it's not stupid anymore :grin:

Reply:
sorry ... totally forgot about it :no_mouth: